src/coreclr/src/nativeaot/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/FieldAccessors/RegularStaticFieldAccessor.cs

```diff
@@ -12,12 +12,14 @@ protected RegularStaticFieldAccessor(IntPtr cctorContext, IntPtr staticsBase, in
             : base(cctorContext, fieldTypeHandle)
         {
             StaticsBase = staticsBase;
-            FieldBase = fieldBase;
+            _fieldBase = fieldBase;
             FieldOffset = fieldOffset;
         }
 
         protected IntPtr StaticsBase { get; }
-        protected FieldTableFlags FieldBase { get; }
+        private readonly FieldTableFlags _fieldBase;
         protected int FieldOffset { get; }
+        protected FieldTableFlags StrippedFieldBase => _fieldBase & FieldTableFlags.StorageClass;
```

Comment:
```suggestion
        protected FieldTableFlags FieldBase => _fieldBase & FieldTableFlags.StorageClass;
```

I would call it this since these are the flags that correspond to the actual base.